pkg/reconciler/workload/apiexportcreate/apiexportcreate_controller.go

```diff
@@ -236,16 +249,18 @@ func (c *controller) process(ctx context.Context, key string) error {
 	if err != nil && !apierrors.IsNotFound(err) {
 		return err
 	} else if apierrors.IsNotFound(err) {
-		klog.Infof("Creating export %s|%s", clusterName, reconcilerapiexport.TemporaryComputeServiceExportName)
 		export = &apisv1alpha1.APIExport{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: reconcilerapiexport.TemporaryComputeServiceExportName,
+				Name:        reconcilerapiexport.TemporaryComputeServiceExportName,
+				Annotations: map[string]string{logicalcluster.AnnotationKey: clusterName.String()},
```

Comment:
This is I guess not the best thing ever, but it means we can use simple logging helpers to attach this object to the context, or we can proliferate a `FromWithWorkspace(obj, ws)` ...

Reply:
I think I would prefer `FromWithWorkspace`, or `WithObject(logger, export).WithValues("workspace",, clusterName.String())`